tests/behaviour/behaviour.test.ts

```diff
@@ -33,16 +33,18 @@ interface AsyncTestCluster {
   dependencies: Map<string, string[]>;
 }
 
-const accountDir = '.starknet_accounts_devnet';
-fs.mkdirSync(accountDir, { recursive: true });
-fs.writeFileSync(path.join(BASE_PATH, accountDir, 'starknet_open_zeppelin_accounts.json'), '{}');
-const address = execSync(`${starknetCliCall('new_account', '')} | awk 'NR==1 {printf $3}'`);
+fs.writeFileSync(path.join(BASE_PATH, 'tests', 'cli', 'starknet_open_zeppelin_accounts.json'), '{}');
+const new_account = `${starknetCliCall('new_account', '')} | awk 'NR==1 {printf $3}'`;
+console.log(new_account);
```

Comment:
Do we want that? If we show it for user information, do we want to add some text explaining what it is?